### PR TITLE
fix create-hypertable-old links

### PR DESCRIPTION
**Pull request by alejandrodnm**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/hypertables/create.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ keywords: [hypertables, create]
 <Highlight type="note">
 This code uses the new generalized hypertable API introduced in
 TimescaleDB 2.13. The [old interface for `create_hypertable is also
```
Comment (JamesGuthrie) [inline use-timescale/hypertables/create.md]:
```suggestion
TimescaleDB 2.13. The [old interface for `create_hypertable` is also
```

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

File: use-timescale/hypertables/create.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ keywords: [hypertables, create]
 
 <Highlight type="note">
 This code uses the new generalized hypertable API introduced in
-TimescaleDB 2.13. The [old interface for `create_hypertable is also
-available](create_hypertable_old.md).
+TimescaleDB 2.13. The [old interface for `create_hypertable` is also
+available][create-hypertable-old].
```
Comment (JamesGuthrie) [inline use-timescale/hypertables/create.md]: This link is still broken. I think these types of links don't work in Highlight blocks.
```suggestion
available](/api/:currentVersion:/hypertable/create_hypertable_old/).
```

File: api/create_hypertable.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ see the [hypertable section][hypertable-docs].
 
 <Highlight type="note">
 This reference describes the new generalized hypertable API that was introduced in TimescaleDB 2.13.
-The [old interface for `create_hypertable is also available][create-hypertable-old].
+The [old interface for `create_hypertable` is also available][create-hypertable-old].
```
Comment (JamesGuthrie) [inline api/create_hypertable.md]: This link is still broken. I think these types of links don't work in Highlight blocks.

```suggestion
The [old interface for `create_hypertable` is also available](/api/:currentVersion:/hypertable/create_hypertable_old/).
```

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

**Review by JamesGuthrie — APPROVED** [review]
The link is still broken in the preview, but I suspect that's because of something about the preview environment and not the docs change itself.

File: api/create_hypertable.md [hunk 4]
```diff
@@ -139,6 +139,6 @@ SELECT create_hypertable('events', by_range('event', time_partitioning_func => '
 [create_distributed_hypertable]: /api/:currentVersion:/distributed-hypertables/create_distributed_hypertable
 [hash-partitions]: /use-timescale/:currentVersion:/hypertables/about-hypertables/#hypertable-partitioning
 [hypertable-docs]: /use-timescale/:currentVersion:/hypertables/
-[create-hypertable-old]: /api/:currentVersion:/hypertable/create_hypertable_old
+[create-hypertable-old]: /api/:currentVersion:/hypertable/create_hypertable_old/
```
Comment (JamesGuthrie) [inline api/create_hypertable.md]: I suspect that this is now no longer used (you should check).
```suggestion
```

File: use-timescale/hypertables/create.md [hunk 5]
```diff
@@ -69,3 +69,4 @@ data. For more information about migrating data, see the
 [install]: /getting-started/latest/
 [postgres-createtable]: https://www.postgresql.org/docs/current/sql-createtable.html
 [postgresql-timestamp]: https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_timestamp_.28without_time_zone.29
+[create-hypertable-old]: /api/:currentVersion:/hypertable/create_hypertable_old/
```
Comment (JamesGuthrie) [inline use-timescale/hypertables/create.md]: Pretty sure this is unnecessary now.
```suggestion
```

**Comment by alejandrodnm** [comment]
YOLO 🚢

**Pull request merged**